### PR TITLE
Display edit on dash

### PR DESCRIPTION
**Pull request by bethanyhaubert**

### CODE REVIEW

**Comment by bethanyhaubert** [comment]
Think i have parent wrong. also need to figure out if i can use 'var client_id = link.dataset.clientId;' in the same way. probably not. The edit button is on the 'view' view, but javascript is displaying 'view' on the dashboard view. is this nesting okay? or should i put the edit button on the dashboard and hide it until the 'view' event is triggered?

File: app/controllers/clients_controller.rb [hunk 1]
```diff
@@ -46,7 +46,7 @@ def add
   def edit_client
     current_client
     current_business
-    render "edit"
+    render "edit",layout: nil
```
Comment (sumeetjain) [inline app/controllers/clients_controller.rb]: Space missing after comma.

**Comment by sumeetjain** [comment]
I _think_ the parent is fine? But I do see that your link doesn't have the `data-client-id` attribute, which the JavaScript you've written for it requires.

**Comment by bethanyhaubert** [comment]
right, where do i need to put that again?

**Comment by sumeetjain** [comment]
On whatever HTML element `link` is referring to.

**Pull request merged**